src/pages/docs/guides/chat/build-task-oriented.mdx

```diff
@@ -0,0 +1,385 @@
+
+**What are task-oriented chats?** Any chat that serves as a companion to a specific activity or workflow: resolving a support ticket, coordinating a delivery, discussing a chess game, collaborating on a project, or facilitating a marketplace transaction. These chats are typically short-lived (minutes to days) and highly contextual. They can be 1:1 or small group conversations.
+
+For large-scale chat scenarios, see our [building livestream chat guide](/docs/guides/chat/build-livestream).
+
+**Task-oriented room architecture:** The optimal approach is one chat room per task, where all participants join to communicate within that specific context. Rooms are:
+
+* **Automatically scaled** - Ably handles any number of concurrent rooms without provisioning
+* **History-enabled** - messages stored for 30 days by default (configurable up to a year)
+* **Kept under control** - [capabilites](/docs/chat/setup#authentication) let you control who can join and the level of access to features and actions
+
+Each room includes all task participants: customers and agents in support, buyers and sellers in marketplaces, players in games, team members in projects. Participants can join and leave as the task evolves. Access to message history and other features can be granted to all or select participants via capabilities.
+
+### Pricing flexibility
+
+Depending on the specifics of your platform, you can choose between per-minute or per-MAU pricing.
+
+Note that one chat room is backed by a single Ably Pub/Sub channel. When the room is active, the channel is active.
+
+With per-minute pricing, you will be charged per message and active room (active channel). This pricing model is ideal if you have a large number of users who might not be active for a long time. Things like deliveries, support tickets, and marketplace transactions are all good examples of this.
+
+With MAU-based pricing you will only be charged for the number of unique active users in the month (and ovarage if you go over the message limits). This pricing model is ideal if your users are very active throughout the month. Collaboration tools and some gaming scenarios are good examples of this. Paying per MAU is typically more predictable than per-minute pricing. Another use case for MAU-based pricing is if you are a platform and you also charge per MAU, to keep costs in sync with your own pricing model.
+
+## Message throughput and rate management
+
+In task-oriented chat, each room is typically low throughput. Scaling comes by running as many rooms as you need at the same time. Ably handles everything from 1:1 conversations and small group discussions to [livestream chats](/docs/guides/chat/build-livestream) with the same reliability and scalability guarantees.
+
+Chat rooms (and Pub/Sub channels) scale horizontally. Ably allows you to have as many rooms as you need running in parallel.
```

Comment:
Do need to call out underlying channels here - are we bringing in the concept too early in the journey / does a reader need to know about them at this point?